dnfdaemon-client/context.hpp

```diff
@@ -92,6 +92,7 @@ class Context {
     std::unique_ptr<libdnf::OptionBool> assume_yes = std::make_unique<libdnf::OptionBool>(false);
     std::unique_ptr<libdnf::OptionBool> assume_no = std::make_unique<libdnf::OptionBool>(false);
     std::unique_ptr<libdnf::OptionBool> allow_erasing = std::make_unique<libdnf::OptionBool>(false);
+    std::unique_ptr<libdnf::OptionBool> skip_broken = std::make_unique<libdnf::OptionBool>(false);
```

Comment:
Do we want to set anything as default? I guess that it is valid for all options.

Reply:
Well, the constructor of the OptionBool requires it. These are command line switches which could be either present - `true`, or not present - `false`. In case the switch is not present, the value is not passed to the server.

Reply:
I tried to contact you on Google chat. We can discuss it directly (chat, meeting).